### PR TITLE
added Hungama.js for support of hungama.com

### PR DESCRIPTION
**Pull request by tanmaysachan**

### CODE REVIEW

**Comment by Coder-256** [comment]
Don't forget to update [`versions.plist`](https://github.com/tanmaysachan/beardedspice/blob/master/BeardedSpice/MediaStrategies/versions.plist), see the [Developers' Guide](https://github.com/beardedspice/beardedspice/blob/master/docs/developers-guide-web.md)

**Comment by tanmaysachan** [comment]
Sorry! I forgot to see that.

File: BeardedSpice/MediaStrategies/Hungama.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	},
+	play: function(){
+		if(document.getElementsByClassName("jp-play")[0].getAttribute("data-tooltip") === "Play")
+			document.getElementsByClassName("jp-play")[0].click();
+	},
+	trackInfo: function() {
+		return {
+			'track': getElementsByClassName("jw-songName")[0].innerText,
+			'album': getElementsByClassName("jw-songAlbum")[0].innerText,
+			'image': getElementsbyClassName("jw-albumThumb")[0].src
```
Comment (Coder-256) [inline BeardedSpice/MediaStrategies/Hungama.js]: Change these to:

- `document.querySelector(".jw-songName").innerText`
- `document.querySelector(".jw-songAlbum").innerText`
- `document.querySelector(".jw-albumThumb").src`

You have to include `document.` before each of those functions, `querySelector(".class")` is faster than `document.getElementsByClassName("class")[0]`, and also these are capitalized correctly (the "b" in `getElementsbyClassName` was not capitalized for the image).

Comment (tanmaysachan) [inline BeardedSpice/MediaStrategies/Hungama.js]: Oh, I can't believe how I missed that, this is my first time, so stuff is going all over the place..
Thanks for being helpful!

**Comment by 1ps0** [comment]
:+1:

**Comment by Stillness-2** [comment]
What about readme.md? Why nobody don't add new strategy in description? :)

**Comment by Coder-256** [comment]
Yes, I forgot about that. Also [beardedspice.github.io](https://beardedspice.github.io/), but that must be done in a different pull request (maybe it should be pulled automatically from this repo via javascript?)

**Pull request merged**